packages/authentication/src/definitions.ts

```diff
@@ -472,6 +474,22 @@ export interface FirebaseAuthenticationPlugin {
   verifyBeforeUpdateEmail(
     options: VerifyBeforeUpdateEmailOptions,
   ): Promise<void>;
+  /**
+   * Checks the current status of app tracking transparency.
+   *
+   * Only available on iOS.
+   *
+   * @since 7.2.0
+   */
+  checkAppTrackingTransparencyPermission(): Promise<CheckAppTrackingTransparencyPermissionResult>;
+  /**
+   * Opens the system dialog to authorize app tracking transparency.
+   *
+   * Only available on iOS.
+   *
+   * @since 7.2.0
+   */
+  requestAppTrackingTransparencyPermission(): Promise<void>;
```

Comment:
Please also return a result here so that we do not have to make another `check(...)` method call.